__tests__/utils.ts

```diff
@@ -0,0 +1,392 @@
+import { TaxBaseSubscriptionFragment } from "@/generated/graphql";
+import { createMocks, RequestMethod } from "node-mocks-http";
+import { AvataxConfig, FetchTaxesPayload } from "../backend/types";
+import { DocumentType } from "ava-typescript/lib/enums/DocumentType";
+import { TransactionModel } from "ava-typescript/models";
+import { BoundaryLevel } from "ava-typescript/lib/enums/BoundaryLevel";
+import { DocumentStatus } from "ava-typescript/lib/enums/DocumentStatus";
+import { JurisdictionType } from "ava-typescript/lib/enums/JurisdictionType";
+import { RateType } from "ava-typescript/lib/enums/RateType";
+
+type RequestConfiguration = {
+  method: RequestMethod;
+  event?: string;
+  domain?: string;
+  signature?: string;
+};
+export const mockRequest = (configuration: RequestConfiguration) => {
+  const method = configuration.method;
+  const { req, res } = createMocks({ method });
+  req.headers = {
+    "content-type": "application/json",
+  };
+  if (configuration.domain !== undefined) {
+    req.headers["saleor-domain"] = configuration.domain;
+  }
+  if (configuration.event !== undefined) {
+    req.headers["saleor-event"] = configuration.event;
+  }
+  if (configuration.signature !== undefined) {
+    req.headers["saleor-signature"] = configuration.signature;
+  }
+
+  return { req, res };
+};
+export const dummyCalculateTaxesPayloadForCheckout: TaxBaseSubscriptionFragment =
```

Comment:
imho those fixtures should be stored in a separate file because they're really long 😅